site/sitemap.xml

```diff
@@ -660,27 +668,27 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>https://spark.apache.org/mllib/</loc>
+  <loc>https://spark.apache.org/sql/</loc>
```

Comment:
I'm not sure why this keeps changing order. It doesn't matter, so don't spend time on it, but do you have the latest jekyll etc?

Reply:
Hmm, that I don't know. I did try to update jekyll + all other packages etc before running this.
I could just reverse this hunk, which is simple to do...